src/client.cpp

```diff
@@ -22,10 +22,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <sstream>
 #include <IFileSystem.h>
 #include "jthread/jmutexautolock.h"
+#include "util/auth.h"
```

Comment:
Should auth.cpp/h and srp.c/h really be in util?? They do take on larger roles than utility functions, you know.